src/common/except.h

```diff
@@ -143,6 +143,10 @@ struct not_supported : virtual user_error
 {
 };
 
+struct expected_error : virtual caspar_exception
```

Comment:
Please, no

Reply:
How would you rather this was handled?

Reply:
Nevermind, I forgot that the amcp handlers add the return code themselves, so I don't need to throw an exception to still report a failure code.